src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const filterByInput = (input: string) => {
+  return moviesFromServer.filter(movie => {
+    const title = movie.title.toLowerCase();
+    const description = movie.description.toLowerCase();
+
+    return title.match(input) || description.match(input);
```

Comment:
I think the **match** method is not needed here, because the **match()** method retrieves the result of matching a string against a regular expression, but our input is not a regular expression, so **includes()** would be more appropriate.

Reply:
includes might to be better, but matrch returns an Array containing the results of the match, or null if nothing was matched, so result will be same